model-server/src/test/kotlin/org/modelix/model/server/handlers/RepositoriesManagerTest.kt

```diff
@@ -0,0 +1,62 @@
+package org.modelix.model.server.handlers
+
+import kotlinx.coroutines.runBlocking
+import kotlinx.coroutines.test.runTest
+import org.modelix.model.lazy.RepositoryId
+import org.modelix.model.server.store.IgniteStoreClient
+import org.modelix.model.server.store.InMemoryStoreClient
+import org.modelix.model.server.store.IsolatingStore
+import org.modelix.model.server.store.LocalModelClient
+import kotlin.test.AfterTest
+import kotlin.test.Test
+import kotlin.test.assertEquals
+import kotlin.test.assertTrue
+
+class RepositoriesManagerWithInMemoryStoreClientTest : RepositoriesManagerTest(InMemoryStoreClient())
+class RepositoriesManagerWithIgniteStoreClientTest : RepositoriesManagerTest(IgniteStoreClient(inmemory = true))
+
+abstract class RepositoriesManagerTest(val store: IsolatingStore) {
+    protected val repoManager = RepositoriesManager(LocalModelClient(store))
+    protected fun initRepository(repoId: RepositoryId) {
+        runBlocking {
+            repoManager.createRepository(repoId, "testUser", useRoleIds = true, legacyGlobalStorage = false)
+        }
+    }
```

Comment:
This could simply be a suspend function as it is always called from a coroutine?

Reply:
fixed